### DISCUSSION TITLE
.NET MAUI MSIX icon background and shortcut tile

### DISCUSSION THREAD

**Discussion by Nav-Pandher** [CommunityToolkit/Maui#1923] [Q&A] [answered]
When i have built the MSIX file from VS2022 and click on it to install the application, in the top right hand corner the icon has a background sitting around my icon.

Also when I create a shortcut on the desktop, it also has a tile background.

For example, in the screenshot below, I've used a test icon which has a transparent background, but it shows within a blue square.

![image](https://github.com/CommunityToolkit/Maui/assets/114393351/4a2d5db3-fa1e-4ea7-a421-186b2728d6aa)

Then for the shortcut, you can see in the screenshot below that it has a blue tile around the icon. (the black colour is just my desktop background)

![image](https://github.com/CommunityToolkit/Maui/assets/114393351/beb42bdd-68be-40a7-8037-08982666c808)

In the start menu and the taskbar, the app shows correctly with a transparent background:
![image](https://github.com/CommunityToolkit/Maui/assets/114393351/933b540b-0e2f-4046-8605-e12182108828)

I need the MSIX installer to just show my icon with no background like many other apps, see the example below:

![image](https://github.com/CommunityToolkit/Maui/assets/114393351/5d848ee2-5e61-4233-8139-379619e3ec14)

**Comment by TheCodeTraveler** [comment, marked as answer]
Hi @Nav-Pandher! I think you meant to open this Discussion on the .NET MAUI GitHub repo: https://github.com/dotnet/maui

You're currently in the .NET MAUI Community Toolkit GitHub repo.

**Reply by Nav-Pandher** [reply]
Hi Brminnick, 

Sorry about that, I'll raise it on the correct repo, thanks

**Comment by CliffAgius** [comment]
Closed as Answered.
